style.css

```diff
@@ -219,3 +218,133 @@ nav li {
   background-position: right bottom;
   background-size: 32vw;
 }
+
+.contact {
```

Comment:
- This element should align its elements to the center. Kindly see the screenshot image of your page and implement changes. 
![Screenshot this](https://user-images.githubusercontent.com/64894927/121272481-c04caa00-c8ce-11eb-8f42-32b7ab624d73.png)